### PR TITLE
AlertLogic pack

### PR DESCRIPTION
**Pull request by jjm**
### Status 

- new pack, work in progress (incomplete)

### Description

Pack for interacting with AlertLogic secuirty service over the ActiveIntergration API. Mainly focused on the ScanAPI in the first instance.

### Checklist (tick everything that applies)

- [x] Metadata: pack.yaml, icon, structure (required for new packs)
- [x] Version bump (required for changed packs)
- [x] Code linting (required, can be done after the PR checks)
- [ ] Tests (not required but really recommended)

### CODE REVIEW

**Comment by jjm** [comment]
This is an very early work in progress, but I was looking to get some linting feedback....

**Comment by jjm** [comment]
This has some commits another branch as I did not branch from master. I'll need to reverse those I think...

File: packs/alertlogic/aliases/list_scan_results.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ack:
+  enabled: true
+  append_url: false
+  format: "Retrieving scan results..."
+result:
+  format: |
+    {% if execution.status == 'succeeded' %}
+    Scan _{{execution.parameters.scan_title}}_ found *{{execution.result.result.total_vulns}}* vulnerabilities on *{{execution.result.result.total_hosts}}* host(s){~}
+
+    *Vulnerability Summary:* {% if execution.result.result.vulns_summary.Urgent != 0 %}_Urgent:_ {{execution.result.result.vulns_summary.Urgent}}; {% endif %}
```
Comment (emedvedev) [inline packs/alertlogic/aliases/list_scan_results.yaml]: A tiny format suggestion: a newline before "urgent" maybe?

File: packs/alertlogic/pack.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: alertlogic
+description: AlertLogic ActiveIntegration APIs 
+keywords:
+    - AlertLogic
+version: 0.1
+author: st2-dev
+email: info@stackstorm.com
```
Comment (emedvedev) [inline packs/alertlogic/pack.yaml]: You're welcome to mention yourself or Pulsant as the author, of course

**Comment by emedvedev** [comment]
> packs/alertlogic/icon.png
The icon should be 64x64 in size: that's the size we currently use in Web UI and the list in this repo, so a bigger icon might break something. It would also be better if there's no small text on it, because it'll look ugly on resizing.

Aside from a few very minor suggestions and the icon format, the contribution is awesome, detailed and thoughtful. Great work.

**Comment by jjm** [comment]
Thanks for the comments, I'll get them addressed tomorrow.

**Comment by jjm** [comment]
I've found a icon with no text and made it the right size. Turns out the other pack I'm working on had the right sized icon and this one had not been updated!

**Comment by emedvedev** [comment]
Great! Do you plan to improve anything else (the pack is still marked WIP) or can I merge?

**Comment by jjm** [comment]
I'm going to have a look over the other suggestions now and check up on the pack author. Anything else I can add as different PR.

**Comment by jjm** [comment]
I've corrected a small issue with one of the aliases, I think we are good to merge (assuming my changes passes CI).

I'll have a think about testing, but it would be all mocking due to the external service dependencies.

**Comment by jjm** [comment]
Hopefully this CI should pass!

**Comment by jjm** [comment]
@emedvedev Thanks for the comments, I think we are good to merge.

**Comment by emedvedev** [comment]
Alright! Merging, and thanks a lot for the contribution.

**Pull request merged**